docs/Learning-Environment-Create-New.md

```diff
@@ -472,3 +472,20 @@ RollerBall environment:
    your Assets browser, turning it into a prefab.
 1. You can now instantiate copies of the TrainingArea prefab. Drag them into
    your scene, positioning them so that they do not overlap.
+
+## Optional: Training Using Concurrent Unity Instances
+In order to run concurrent Unity instances during training, set the number of environment instances using the command line option ```--num-envs=<n>``` when you invoke mlagents-learn. Optionally, you can also set the ```--base-port```, which is the starting port used for the concurrent Unity instances.
+
+For example,
+
+```
+mlagents-learn config/rollerball_config.yaml --run-id=RollerBall --num-envs=2
+```
+
+will start ML Agents training with two environment instances. Combining multiple training areas withint the same scene, with concurrent Unity instances, effectively gives you two levels of parallelism to speed up training.
+
+Some considerations:
```

Comment:
I would instead point developers here: https://github.com/Unity-Technologies/ml-agents/blob/main/docs/Training-ML-Agents.md#training-using-concurrent-unity-instances

otherwise we'll have similar content in two different pages. Feel free to make updates to that page if you feel it deserves more info

Reply:
Made the updates to address your comments.

Reply:
Done